config/config-contour.yaml

```
@@ -19,6 +19,9 @@ metadata:
   namespace: knative-serving
   labels:
     networking.knative.dev/ingress-provider: contour
+    app.kubernetes.io/name: contour
```

Let's use `net-contour` since this is technically the networking contour plugin
```suggestion
    app.kubernetes.io/name: net-contour
```

---

I'm sensing a theme 😄